src/bio/bio_xstream.c

```diff
@@ -1116,18 +1113,33 @@ bio_nvme_configured(enum smd_dev_type type)
 
 	if (type >= SMD_DEV_TYPE_MAX)
 		return true;
-	/*
-	 * FIXME When storage class 'ram' is used in server yaml, control
-	 * plane regards MD on SSD is enabled and meta role will be set to
-	 * bdev name, that'll fail CI tests before the md on SSD feature is
-	 * fully implemented.
-	 */
-	if (type == SMD_DEV_TYPE_META && !md_on_ssd_enabled)
-		return false;
 
 	return is_role_match(nvme_glb.bd_nvme_roles, dev_type2role(type));
 }
 
+int
+bio_get_lmm_db_path(char *lmm_db_path)
```

Comment:
I think we could avoid exporting such an API from BIO. "lmm_db_path" looks to me is specified by caller.

Reply:
done